### DISCUSSION TITLE
Docusaurus v3: prism HTML script tag highlight

### DISCUSSION THREAD

**Discussion by peterpeterparker** [facebook/docusaurus#9506] [Q&A]
Hi,

I'm trying to migrate to Docusaurus v3 but, I noticed that my blocks of HTML code are rendered without their `<script>` tags being highlighted (see screenshots).

I tried to add "markup" to the `additionalLanguages` but it did not help.

Any idea what's the issue?

---

Example used:

````md
```html title="+layout.svelte"
<script>
  import { onMount } from "svelte";
  import { initJuno } from "@junobuild/core";
</script>
```
````

---

Docusaurus v3

<img width="1536" alt="Capture d’écran 2023-11-04 à 17 47 23" src="https://github.com/facebook/docusaurus/assets/16886711/12db6974-b566-4b94-b724-6dd52f578f0f">

---

Docusaurus v2

<img width="1536" alt="Capture d’écran 2023-11-04 à 17 47 25" src="https://github.com/facebook/docusaurus/assets/16886711/5ae35917-1655-4485-aa25-b442048ce522">

**Comment by Josh-Cena** [comment]
Could you try adding `javascript` to your `additionalLanguages` (and remove `markup`)? My guess is that a recent version of prism-react-renderer messed up and loaded the languages in the wrong order.

**Reply by peterpeterparker** [reply]
Indeed the workaround you are suggesting solves the issue. I'll subscribe to https://github.com/facebook/docusaurus/issues/9517. Thanks for your help @Josh-Cena !

**Comment by Josh-Cena** [comment]
We have an issue now: https://github.com/facebook/docusaurus/issues/9517 so we'll use that instead
